helix-core/src/syntax.rs

```diff
@@ -210,7 +240,7 @@ impl LanguageConfiguration {
                 &injections_query,
                 &locals_query,
             )
-            .unwrap(); // TODO: avoid panic
+            .unwrap_or_else(|query_error| panic!("Could not parse queries for language {:?}. Are your grammars out of sync? Try running 'hx --fetch-grammars' and 'hx --build-grammars'. This query could not be parsed: {:?}", self.language_id, query_error));
```

Comment:
Shouldn't this be `.expect()`?

Reply:
I think I tried with `expect` but the lints wanted `unwrap_or_else` to avoid computing the error message unless it failed